src/components/Tag/index.tsx

```diff
@@ -1,140 +1,147 @@
-import { Theme, css } from '@emotion/react'
+import { Theme, useTheme } from '@emotion/react'
 import styled from '@emotion/styled'
-import PropTypes from 'prop-types'
-import { MouseEventHandler, ReactNode } from 'react'
-import Box, { BoxProps } from '../Box'
-import Icon from '../Icon'
+import { MouseEventHandler, ReactNode, useMemo } from 'react'
+import { Color, SENTIMENTS } from '../../theme'
+import Icon, { IconName } from '../Icon'
 import Loader from '../Loader'
 import Touchable from '../Touchable'
+import Typography from '../Typography'
 
-const disabledStyles = css`
-  opacity: 0.5;
-`
-
-export const variantsContainer = {
-  base: ({ theme }: { theme: Theme }) => css`
-    background-color: ${theme.colors.neutral.backgroundStrong};
-    height: 24px;
-    padding-left: 8px;
-    padding-right: 8px;
-  `,
-  bordered: ({ theme }: { theme: Theme }) => css`
-    padding: 8px;
-    border: 1px solid ${theme.colors.neutral.borderWeak};
-  `,
-}
-
-type TagVariant = keyof typeof variantsContainer
-
-const variantStyles = ({
-  variant,
-  ...props
-}: {
-  variant: TagVariant
-  theme: Theme
-}) => variantsContainer[variant]?.(props)
-
-type StyledContainerProps = {
-  disabled: boolean
-  variant: TagVariant
-} & BoxProps
-
-const StyledContainer = styled(Box, {
-  shouldForwardProp: props =>
-    !['disabled', 'variant'].includes(props.toString()),
-})<StyledContainerProps>`
-  ${({ disabled }) => disabled && disabledStyles}
-  border-radius: 4px;
+const StyledBox = styled('div', {
```

Comment:
Could we use another name than Box, for the sake of the current migration ? :)

Reply:
true

Reply:
done